src/conf_mode/nat.py

```diff
@@ -17,6 +17,7 @@
 import os
 
 from sys import exit
+from pathlib import Path
```

Comment:
Why is the move to pathlib required?

Reply:
When there were firewall.py + nat.py, we use /run/use-vyos-domain-resolver-nat and  /run/use-vyos-domain-resolver-firewall, but now we introduced  /run/use-vyos-domain-resolver-wireguard-wgX, we need to change the file-exist-check in all 3 file.

Code in firewall.py is better than nat.py(using not `Path().glob()`), latest: https://github.com/vyos/vyos-1x/blob/ad1f62688ec1ea7319d7aa0bb62082e553be1863/src/conf_mode/firewall.py#L584

Since we need to do same check in 3 files, I changed the nat.py file.